multirun.py

```diff
@@ -212,8 +212,8 @@ def on_created_or_modified(event):
             index_size = -1
             if isredis:
                 if not args.cluster: # TODO: get total size from all the shards
-                    index_size = redis.ft('ann_benchmark').info()['vector_index_sz_mb']*1000
-                f.attrs["index_size"] = float(index_size)
+                    index_size = float(redis.ft('ann_benchmark').info()['vector_index_sz_mb'])*1000
```

Comment:
nitpicking - shouldn't this be `...*0x1000` (or 1024)

Reply:
:+1: